app/models/katello/repository.rb

```diff
@@ -278,6 +278,10 @@ def group
       all_instances
     end
 
+    def to_hash(content_source = nil)
+      {id: id, name: label, url: full_path(content_source)}
```

Comment:
`name` was previously taken from the repo name - but now it looks at the label. is that intended?

Reply:
@stbenjam was this related to the issue with spaces? not sure if this was the same PR or not

Reply:
Yes, it was intended, anaconda can't take spaces.  I didn't see it used anywhere else - the UI code seems to only be interested in the ID's.

Reply:
We can change the hash to be more correct, and set a label key, if you'd prefer.